src/librustc_trans/mir/mod.rs

```diff
@@ -386,7 +386,7 @@ fn arg_local_refs<'a, 'tcx>(bcx: &Builder<'a, 'tcx>,
 
             let lvalue = LvalueRef::alloca(bcx, arg_ty, &format!("arg{}", arg_index));
             for (i, &tupled_arg_ty) in tupled_arg_tys.iter().enumerate() {
-                let dst = bcx.struct_gep(lvalue.llval, i);
+                let (dst, _) = lvalue.trans_field_ptr(bcx, i);
```

Comment:
Same but here it's less worrying because we *know* it can't be anything other than a tuple, and those don't get packed or anything like that, and `store_fn_arg` doesn't take such an argument.